docs/release-notes.md

```diff
@@ -24,6 +24,7 @@ Packaging changes:
 
 - Require Rust ≥ 1.58.0
 - Add dependency on `zstd` crate and `libzstd` shared library
+- Support `serde_yaml` 0.9
```

Comment:
Maybe silly question, it looks like we used to use serde_yaml "0.9". I see we changed to using bounds which does use "0.9" but the release-notes.md change does not indicate that we used to use "0.9"

Reply:
Note that the original Dependabot commit in this PR bumped us from 0.8.x to 0.9.

Reply:
Ah got you 👍